crates/gpui/src/platform/linux/x11/window.rs

```diff
@@ -403,11 +409,16 @@ impl PlatformWindow for X11Window {
     }
 
     fn content_size(&self) -> Size<Pixels> {
-        self.0.state.borrow_mut().content_size()
+        // We divide by the scale factor here because this value is queried to determine how much to draw,
+        // but it will be multiplied later by the scale to adjust for scaling.
```

Comment:
The comment may not be clear enough, but basically, if we don't do this, window content is cut off :sweat_smile: